### ISSUE TITLE
Length parameter must be greater than 0

### ISSUE THREAD

**Issue by chekalskiy** [mediact/dependency-guard#15]
Getting error after install:

```
In SymbolExtractor.php line 53:

SplFileObject::fread(): Length parameter must be greater than 0
```

**Comment by johmanx10** [comment]
@chekalskiy this bug was introduced in version `1.0.2` and should not be present in earlier versions. I will focus on a patch for now, and meanwhile you could try using an older version.

**Comment by johmanx10** [comment]
@chekalskiy thanks for reporting the bug. It has been patched in version `1.0.3`, which is immediately available for installation.
